gobblin-example/src/main/java/gobblin/example/wikipedia/WikipediaExtractor.java

```diff
@@ -140,6 +144,24 @@ public WikipediaExtractor(WorkUnitState workUnitState) throws IOException {
     this.reader = new WikiResponseReader();
   }
 
+  private String readProp(String key, WorkUnitState workUnitState) {
+    String value = workUnitState.getWorkunit().getProp(key);
+    if (StringUtils.isBlank(value)) {
+      value = workUnitState.getProp(key);
+    }
+    if (StringUtils.isBlank(value)) {
+      value = workUnitState.getJobState().getProp(key);
```

Comment:
As discussed offline, jobState is never null in all practical purposes. We will however still add a check once we take this bug fix to mainstream: https://github.com/linkedin/gobblin/issues/1022